src/Helper/Navigation/PluginManager.php

```diff
@@ -15,15 +16,13 @@
  */
 class PluginManager extends HelperPluginManager
 {
-    /**
-     * @var string Valid instance types.
-     */
     protected $instanceOf = AbstractHelper::class;
 
     /**
      * Default aliases
      *
-     * @var string[]
+     * @var array<string, class-string>
+     * @psalm-var array<array-key, string>
```

Comment:
Keys must be `string` here - can't be `array-key`

Also unsure why the two types are misaligned for `@var` and `@psalm-var`: in both cases, this is `class-string`

Reply:
For the aliases, they are [annotated as string[]](https://github.com/laminas/laminas-servicemanager/blob/3.11.x/src/ServiceManager.php#L75) in SM. I removed @\inheritdoc as per [comment from](https://github.com/laminas/laminas-view/pull/96#discussion_r772277754) @boesing - I've added the correct type _(`array<string, string>`)_ and added the resulting error to the baseline. Does that work? Or would it be better to remove the docs and fix the type in SM?

Reply:
@gsteel would be best to remove the docs here then, I suppose - the type in SM can be refined in a separate patch then

Reply:
Let's leave this as-is for now :)